### DISCUSSION TITLE
Max width property don't work or?

### DISCUSSION THREAD

**Discussion by andreaskviby** [filamentphp/filament#774] [General]
![image](https://user-images.githubusercontent.com/1666004/145213182-ef0e50e5-f69b-4962-a694-55c23fb4a465.png)

In the docs it states that I can set the layouts.max_content_width but the code on how to do that states it is layout and not layouts that should be set in the config file.

This don't seem to work in my filament installation so the annoying table view still has a max width of 1280px. Is there any way to override the .max-w-screen-xl on all the listings without creating my own theme?

**Comment by danharrin** [comment]
Hey, this is probably a bug then, so please open an issue :)

Nice to see you around again, by the way! Hope all is well.

**Comment by vv-ac** [comment]
Hi all,

is it possible to intoduce 8xl, 9xl and so on - sth in between 7xl and full?
In tailwind config I added:
```
extend: {
            spacing: {
                '8xl': '96rem',
                '9xl': '128rem',
            }
...
```

I also published the filament config and then changed the layout => max_content_width to 8xl or 9xl while having npm run watch in the background running.

Frontend did not change though; what am I missing here?

Thanks!

**Reply by danharrin** [reply]
The actual class needs to be present in the Tailwind markup for it to get added to the CSS stylesheet, which is why we don't support custom widths yet

I'd accept a PR that allows you to specify a custom width class, and then you can add the config file to your Tailwind config content paths

**Reply by danharrin** [reply]
#3603
